src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -211,6 +212,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
     @observable searchMutationInput:string;
     @observable viewMutationType:boolean = true;
     @observable viewCopyNumber:boolean = false;
+    @observable viewMutationAndCNA:boolean = false;
```

Comment:
why dont you want to derive this as `@computed`: `viewMutationType && viewCopyNumber`?

Reply:
@adamabeshouse oh I just didn't think of it in the first place. Thanks for pointing it out.